_release/cn/service_center_downloads.md

```diff
@@ -11,7 +11,8 @@ last_modified_at: 2021-06-04T00:50:43-55:00
+| **Apache ServiceComb Service-Center 2.1.0 (LATEST)** | [[src]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-src.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-src.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-src.tar.gz.sha512) | [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-windows-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-windows-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-windows-amd64.tar.gz.sha512) | [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-linux-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-linux-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-linux-amd64.tar.gz.sha512) | [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-darwin-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-darwin-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-darwin-amd64.tar.gz.sha512) | Java-Chassis 1.0.0 or above |
```

Comment:
I just tried to access the url and got this link. https://dlcdn.apache.org/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-src.tar.gz

Reply:
should be https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-src.tar.gz